modules/reference/pages/config/server-configuration-overview.adoc

```diff
@@ -194,6 +195,23 @@ For example, if you enter `${my.env.var}` in the `server.xml` file, it looks for
 . my_env_var
 . MY_ENV_VAR
 
+Variables can be loaded from files that are located in directories that are specified by the `VARIABLE_SOURCE_DIRS` environment variable.
+The default location is `${server.config.dir}/variables`. The name of the file is used as the variable's name, and the contents of the file are used as the variable's value.
```

Comment:
this is a little vague:

the contents of the file are used as the variable's value

what is the file syntax?, how should the "contents" be specified?